src/parser.ts

```diff
@@ -78,6 +172,14 @@ function generateAST(code: string, options: ParserOptions): any {
       extra.jsx = true;
     }
 
+    if (
```

Comment:
I believe this option was recently streamlined to be at the top level, I don't think we should move it back to a nested ecmaFeatures object

Reply:
Good point. Removed.